### PR TITLE
Fixing additional prettier errors

### PR DESCRIPTION
**Pull request by liam-potter**
This PR should fix any issues with Buefy, Vuetify and Tailwind.

### CODE REVIEW

**Comment by liam-potter** [comment]
There will be additional commits coming to further fix issues (now that I know what I am doing 😆).

File: template/frameworks/vuetify/layouts/default.vue [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
           v-for="(item, i) in items"
-          :to="item.to"
           :key="i"
+          :to="item.to"
           router
           exact
         >
           <v-list-tile-action>
-            <v-icon v-html="item.icon" />
+            <v-icon v-html="item.icon" /><% if (eslint === 'yes') { %><!-- eslint-disable-line vue/no-v-html --><% } %>
```
Comment (aldarund) [inline template/frameworks/vuetify/layouts/default.vue]: Dont disable eslint. Rather use {{ }} e.g. ```<v-icon>{{ item.icon }}</v-icon>```

File: template/frameworks/vuetify/layouts/default.vue [hunk 2]
```diff
@@ -34,7 +34,7 @@
         icon
         @click.stop="miniVariant = !miniVariant"
       >
-        <v-icon v-html="miniVariant ? 'chevron_right' : 'chevron_left'" />
+        <v-icon v-html="miniVariant ? 'chevron_right' : 'chevron_left'" /><% if (eslint === 'yes') { %><!-- eslint-disable-line vue/no-v-html --><% } %>
```
Comment (aldarund) [inline template/frameworks/vuetify/layouts/default.vue]: Dont disable eslint. Rather use {{ }}

File: template/frameworks/buefy/pages/index.vue [hunk 3]
```diff
@@ -16,13 +17,16 @@
...
             </div>
           </div>
-          <footer class="card-footer">
-            <div
+          <footer class="card-footer"><% if (eslint === 'yes') { %>
+            <!-- eslint-disable vue/no-v-html -->
+            <% } %><div
               class="card-footer-item"
-              v-html="feature.content"/>
+              v-html="feature.content"
```
Comment (aldarund) [inline template/frameworks/buefy/pages/index.vue]: This cycle doesnt make much sense. In original it just inlined and its fine. Lets do same
https://github.com/buefy/buefy/blob/dev/docs/pages/Home.vue

Comment (liam-potter) [reply template/frameworks/buefy/pages/index.vue]: Fixed - separated repeatable parts to an individual `Card` component for sake of DRY.

File: template/frameworks/vuetify/layouts/default.vue (hunk 2 above)
Comment (liam-potter) [reply template/frameworks/vuetify/layouts/default.vue]: Fixed

File: template/frameworks/vuetify/layouts/default.vue (hunk 1 above)
Comment (liam-potter) [reply template/frameworks/vuetify/layouts/default.vue]: Fixed

**Pull request merged**